### ISSUE TITLE
Add simple app/script to change governor in fly

### ISSUE THREAD

**Issue by AngryPenguinPL** [OpenMandrivaAssociation/OpenMandrivaAssociation.github.io#39]
It is worth considering adding a simple script or application to change the cpu governor on the fly.

There is an applet for plasma e.g. plasma pstate: https://github.com/jsalatas/plasma-pstate

Wdyt about packaging it and import to iso?

Depends on: https://github.com/OpenMandrivaAssociation/OpenMandrivaAssociation.github.io/issues/38

**Comment by abucodonosor** [comment]
Nope, please see what it does.

Also note it does add this:

https://raw.githubusercontent.com/jsalatas/plasma-pstate/master/99-plasma-pstate

Which is a no-no-no-no :-).

Also the script itself is buggy and missing things or assuming some things. I guess is fine for personal usage but to package it need be fix up first.

**Comment by AngryPenguinPL** [comment]
> Nope, please see what it does.

Ok. So this one is not suitable.
By the way, it would be nice to have something similar (but working well) in the system.

**Comment by tpgxyz** [comment]
@AngryPenguinPL i'd go with packaging that plasma widget. Maybe we could help upstream with feedback and getting it fixed.

**Comment by abucodonosor** [comment]
@tpgxyz 

You want to package something want to run sudo with NOPASS on a buggy bash script?
Seriously?

You don't help upstream with that, you can help upstream by fixing the thing, and send patches in. And fixing before packaging ofc.

**Comment by tpgxyz** [comment]
@abucodonosor  why not? Just to give it a try.
